### DISCUSSION TITLE
Firebase not initialized in flutter module embeded in iOS native project

### DISCUSSION THREAD

**Discussion by grisvladko** [firebase/firebase-ios-sdk#10671] [Q&A]
Hello
quick overview of the project : 
a native iOS app with Flutter embedded as module.
problem: 
we are unable to catch crashes from flutter with Firebase Crashlytics.
when we try to initialize Firebase in the main.dart file in flutter, we get Firebase not initialized.
all the tutorials teach us how to use Firebase in Flutter but as the main project. how do you use it when flutter is embeded as module?
Thank you for your help.

**Comment by paulb777** [comment]
@grisvladko You may have better luck with your question on a Flutter support channel like stackoverflow or one of the flutter GitHub repositories.

**Comment by ahmdmau** [comment]
I had same issue here
